console/console-init/ui/src/components/common/NetworkStatusAlert.tsx

```diff
@@ -26,6 +26,11 @@ const NetworkStatusAlert: React.FunctionComponent = () => {
     hasNetworkError !== undefined && setAlertVisible(hasNetworkError);
   }, [hasNetworkError]);
 
+  const errorMessage =
+    statusCode && statusCode === 403
```

Comment:
can we create constants for error code in constant file. you should create a enum object just like we have FetchPolicy and use it.
enum ServerErrors{
  ERROR_403:403,
  ERROR_401:401
}

Reply:
Fixed